### DISCUSSION TITLE
Microservices and Orchestration

### DISCUSSION THREAD

**Discussion by Lewisrxliu** [RoadSideUniversity/ClubHouse2024#20] [Ideas]
Microservices and Orkes Conductor Meetup Singapore. Join me today. 5:00-8:00PM online
https://www.linkedin.com/events/microservicesandorkesconductorm7119604302511316992/

Invite from Orkes

After the success of our first meetup in Singapore which brought together 90+ technical professionals over exciting conversations around Microservices and Orchestration, we're thrilled to be back in this tech hub of Asia. 🇸🇬 🇸🇬 

This time we will not only be highlighting the challenges and experiences of developing such complex systems but also share real-world example stories and dive into topics such as AI orchestration, workflow orchestration, developer-centric scalability etc. 

Reserve your spot immediately as we have limited spots for the event -https://lnkd.in/gt5gGmKN

**Comment by guttasingapore** [comment]
原来他们在这么玩, 我错过了免费的晚餐@AWS Manulife， 节省了3个小时
https://orkes.io/meetups/microservices-and-orkes-conductor-meetup-singapore
https://github.com/Netflix/conductor
[[Slack Community](https://join.slack.com/t/orkes-conductor/shared_invite/zt-xyxqyseb-YZ3hwwAgHJH97bsrYRnSZg)](https://app.slack.com/client/T02KG20GJ1Z/C02K1B5PM0X)
[GitHub Discussion Forum](https://github.com/Netflix/conductor/discussions)
AWS/AIA/Orkes - Git/Netflix/conductor/Slack, Application Orchestration with AI and Human tasks, 
自学成果和心得：
Conductor is a platform created by Netflix to orchestrate workflows that span across microservices. Conductor is maintained by Media Workflow Infrastructure team at Netflix. 原来是基于GitHub的, 第一次pull request, create codespace in Github, done
MS extension pack for Java installed, 原来是基于Redhat的
